src/Middleware/RecordTransaction.php

```diff
@@ -66,7 +66,7 @@ public function handle($request, Closure $next)
             $transaction->setTransactionName($this->getRouteUriTransactionName($request));
         }
 
-        $transaction->stop($this->timer->getElapsedInMilliseconds());
```

Comment:
The use of the Timer to get the transaction duration should be retained.

Reply:
Huh, now I see what's wrong with agent version 6.5.2, it was missing timer constructor and it was throwing here timer not started. that's why I've removed it, but now it doesn't calculate duration from the time request came, but when it activated middleware?

Might be good to start tracking changes  of agent in some changelog though :)